src/components/ec-btn-dropdown/ec-btn-dropdown.vue

```diff
@@ -99,40 +124,62 @@ export default {
 </script>
 
 <style>
-  .ec-btn-dropdown {
-    @apply tw-inline-flex;
-
-    max-width: 246px;
-
-    &__dropdown-btn {
-      @apply tw-rounded-l-none;
-      @apply tw-border-gray-6 tw-border-l tw-border-solid;
-      @apply tw-bg-gray-7;
-
-      &--is-open {
-        @apply tw-bg-key-4;
-        @apply tw-border-key-4;
-        @apply tw-text-gray-8;
-      }
-
-      &:hover {
-        @apply tw-border-key-3;
-        @apply tw-bg-key-3;
-      }
-
-      &:disabled {
-        @apply tw-text-gray-5;
-      }
+.ec-btn-dropdown {
```

Comment:
We should revert changes that have been done by the prettier. It's hard to tell here what has changed.